index.html

```diff
@@ -12,44 +12,171 @@
+                    <ul class="tools">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                    <a class="button" href="">See Project</a>
+                </div>
+            </div> 
+         </section> 
+
+         <section class="about">
             <div class="intro">
-                <h1>I’m Alexandra <br>Glad to see you!</h1>
+                <h1>About<br>Myself</h1>
                 <p>I can help you build a product , feature or website Look through some of my work and experience! If
                     you like what you see and have a project you need coded,
-                    don’t hestiate to contact me.</p>
+                    don’t hestiate to contact me.
+                </p>
             </div>
-            <!-- social medials lins -->
+                <!-- social medials lins -->
             <div class="social-medials">
                 <p>LET’S CONNECT</p>
                 <ul class="social-icons">
-                    <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
                     <li><a href="https://github.com/Ridwanullahi-code"><img src="images/github.png" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
                     <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Linkedin.png" alt=""></a></li>
+                    <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
                     <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
+                    
                 </ul>      
             </div>
-        </section>
-    </main>
-
+            <!-- button to download resume -->
+            <a class="resume-button" href="">Get my resume</a>
+            <!-- area-->
+            <div class="technologies">
+                <h2>Languages</h2>
+                <div>
+                    <img src="images/Ellipse1.png" alt="">
+                    <p>Javascript</p>
+                </div>
+                <div>
+                    <img src="images/Ellipse2.png" alt="">
+                    <p>HTML</p>
+                </div>
+                <div>
+                    <img src="images/Ellipse3.png" alt="">
+                    <p>CSS</p>
+                </div>
+            </div>
+            <!-- kist of hard skills -->
+            <div class="skills">
+                <div>
+                    <h2>Frameworks</h2>
+                    <p>></p>
+                </div>
+                <hr>
+                <div>
+                    <h2>Skills</h2>
+                    <p>></p>
```

Comment:
- [ ] As demanded by the project requirements. Please use `<ul>` and `<li>` for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

- [ ] Please kindly use indentation of two spaces in all your documents. This is a convention and [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).